flights/berks_ceh_conditions_flight.yaml

```diff
@@ -8,6 +10,15 @@ entityDefinitions:
       nc.SubjectIdentification:
         type: "nc.SubjectIdentification"
         column: "PersonalID"
+    ol.datasource:
```

Comment:
We should add datasource to every single flight where people appear, but make sure to add conditions to it also.